src/SDKs/SqlManagement/Sql.Tests/Utilities/SqlManagementTestUtilities.cs

```diff
@@ -287,8 +287,15 @@ public static void ValidateVirtualNetworkRule(VirtualNetworkRule expected, Virtu
             Assert.NotNull(actual.Id);
             Assert.Equal(expected.VirtualNetworkSubnetId, actual.VirtualNetworkSubnetId);
         }
+        
+        public static void ValidateServerDnsAlias(ServerDnsAlias actual, string name)
```

Comment:
I understand you're following the pattern here, but I think it would be better in your DNS alias test class